### ISSUE TITLE
May I ask how to use this code?

### ISSUE THREAD

**Issue by Faust-Wang** [noether/uavitl#1]

**Comment by noether** [comment]
> 

Sure, I should add a short description on how to use it with XPlane 9. 

Still interested?

**Comment by Faust-Wang** [comment]
> > 
> 
> Sure, I should add a short description on how to use it with XPlane 9. 
> 
> Still interested?


Of course, your work is fascinating, but I don't know how to use your code. Could you add a use video or use documentation

**Comment by noether** [comment]
Thanks for your kind words.

Sure, I will add a short guide (last time I explained it to somebody else was a couple of lines), hopefully, this weekend, so you can have a rotorcraft (or some of them) in XPlane. I will keep you posted it here once it is ready.

**Comment by Faust-Wang** [comment]
> Thanks for your kind words.
> 
> 
> 
> Sure, I will add a short guide (last time I explained it to somebody else was a couple of lines), hopefully, this weekend, so you can have a rotorcraft (or some of them) in XPlane. I will keep you posted it here once it is ready.

Thank you very much for your help.

**Comment by noether** [comment]
I did not forget about it, just no time to seat down, install xplane, and run the code again.
